birdhouse/config/canarie-api/docker_configuration.py.template

```diff
@@ -242,7 +242,7 @@ PLATFORMS = {
         'info': {
             'name': 'PAVICS',
             'synopsis': 'The PAVICS (Power Analytics for Visualization of Climate Science) platform is a collection of climate analysis services served through Open Geospatial Consortium (OGC) protocols. These services include data access, processing and visualization. Both data and algorithms can be accessed either programmatically, through OGC-compliant clients such as QGIS or ArcGIS, or a custom web interface.',
-            'version': '1.21.1',
+            'version': '1.22.0',
             'releaseTime': '2022-08-24T15:43:51Z',
```

Comment:
@fmigneault release time did not update.  Is there a special option we need to run `bumpversion` with, for release time to update?  If yes, we need to doc this special option.  I just checked one of my previous release, the release time did not update for me as well.  Or need a special version of `bumpversion`?

Reply:
Seems like the portion https://github.com/bird-house/birdhouse-deploy/blob/master/.bumpversion.cfg#L28-L29 cannot update itself because bumpversion rewrites the `.bumpversion.cfg` file after applying changes using what it initially read. So the update works the first time when datetime match, but fail on following cases because the new datetime was not applied. 

I think we could hack around this using a separate file that contains only the release time. Bumpversion could modify that file without problem. It would need to be mounted in the docker image for the python config to read it.